src/databricks/labs/lsql/dashboards.py

```diff
@@ -355,9 +355,30 @@ class MarkdownTile(Tile):
     _position: Position = Position(0, 0, _MAXIMUM_DASHBOARD_WIDTH, 3)
 
 
-def replace_database_in_query(node: sqlglot.Expression, *, database: str) -> sqlglot.Expression:
-    """Replace the database in a query."""
-    if isinstance(node, sqlglot.exp.Table) and node.args.get("db") is not None:
+def replace_database_in_query(
```

Comment:
rewrite this as a public method on `DashboardMetadata`: `def replace_database(self, catalog_name: str, schema_name: str) -> DashboardMetadata:`, which would create a copy of `DashboardMetadata`, but with all of the queries rewritten with the target catalog/db